hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -71,18 +71,18 @@
 
   public static final String PENDING_COMPACTION_OPS = String.format("%s/%s", BASE_URL, "compactions/pending/");
 
-  public static final String LATEST_PARTITION_DATA_FILES_URL =
-      String.format("%s/%s", BASE_URL, "datafiles/latest/partition");
-  public static final String LATEST_PARTITION_DATA_FILE_URL =
-      String.format("%s/%s", BASE_URL, "datafile/latest/partition");
-  public static final String ALL_DATA_FILES = String.format("%s/%s", BASE_URL, "datafiles/all");
-  public static final String LATEST_ALL_DATA_FILES = String.format("%s/%s", BASE_URL, "datafiles/all/latest/");
-  public static final String LATEST_DATA_FILE_ON_INSTANT_URL = String.format("%s/%s", BASE_URL, "datafile/on/latest/");
-
-  public static final String LATEST_DATA_FILES_RANGE_INSTANT_URL =
-      String.format("%s/%s", BASE_URL, "datafiles/range/latest/");
-  public static final String LATEST_DATA_FILES_BEFORE_ON_INSTANT_URL =
-      String.format("%s/%s", BASE_URL, "datafiles/beforeoron/latest/");
+  public static final String LATEST_PARTITION_BASE_FILES_URL =
+      String.format("%s/%s", BASE_URL, "basefiles/latest/partition");
```

Comment:
since a job is deployed in one shot, this may be ok.. no need to support backwards compatibility of endpoints etc

Reply:
Yes.